webpack.config.js

```diff
@@ -1,12 +1,19 @@
 // From https://github.com/mermaidjs/mermaid-webpack-demo/
 
+'use strict';
+
+const path = require('path')
+
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const TerserPlugin = require('terser-webpack-plugin');
 
 const config = {
   target: 'web',
   entry: {
-    'page': './page.js'
+    'page': [
+      './page.js',
+      './index.js'
```

Comment:
Not sure about this change. We webpack the code loaded inside the markdown preview, however `index.js` is the main extension code

Reply:
Well, I'm not a webpack expert, but I stumbled into an error in the console early on when debugging that said the `require('markdown-it-container')` call failed due to unknown package, so I poked around and at least according to `npx webpack --mode production --display-modules` it doesn't look like it's bundling it without the extra entry point being listed:

```console
$ npx webpack --mode production --display-modules
...
    [369] ./node_modules/moment-mini/moment.min.js 52.1 KiB {0} [built]
    [370] ./node_modules/moment-mini/locale sync ^\.\/.*$ 189 bytes {0} [optional] [built]
    [371] ./node_modules/stylis/stylis.js 39.9 KiB {0} [built]
    [372] ./index.js 1.78 KiB {0} [built]
    [373] ./node_modules/markdown-it-container/index.js 4.2 KiB {0} [built]  <-- MISSING without index.js entry point

    WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
    This can impact web performance.
...
```

I'm not sure where else webpack would be getting it from otherwise, as `page.js` doesn't reference it.

Reply:
How about we remove the webpack changes from this PR and I can take a look separately to see if they are required

Reply:
I rebased to clean out the webpack change. I can't seem to get the error to repro again, so unfortunately I can't give you any more context to track it down separately. If it crops up again, I'll file a separate issue.